### DISCUSSION TITLE
Why require docker?

### DISCUSSION THREAD

**Discussion by jult** [grafolean/grafolean#50] [General]
Honestly, looks like overkill and more complex for this purpose. I would have to install docker just for this, on a debian router, which is already living as a clone/cluster, with freeNGINX on it. It should just as well be able to run on the host without docker. Plus, this way you would have to do network transports for the flow (export/import) and are actually introducing extra points of failure.
Other than that it is exactly what I would need.

**Comment by grafolean** [comment]
Hey, no problem, you can easily install it without Docker. It's just that it is easier for me, personally, to provide a set of images (and Dockerfiles for them) which work, instead of supporting many many *many* different kinds of installations. However Dockerfiles are basically just a set of commands you need to run on the system to get it working, so nothing is stopping you from just following them, or in your case, to adapt the procedure to use your existing nginx installation.

I agree with netflow, that workaround (using a shared named pipe between containers to avoid Docker networking bugs) adds complexity where it should not be needed. It's a tradeoff for reliable installations.

If you install it, please share the experience. I'm curious how it goes. :)
